README.md

```diff
@@ -88,6 +88,80 @@ The Lumigo OpenTelemetry Distro for Java additionally supports the following con
 
 For more configuration options, see the [Upstream Agent Configuration](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/).
 
+### Execution Tags
+
+[Execution Tags](https://docs.lumigo.io/docs/execution-tags) allow you to dynamically add dimensions to your invocations so that they can be identified, searched for, and filtered in Lumigo.
+For example: in multi-tenanted systems, execution tags are often used to mark with the identifiers of the end-users that trigger them for analysis (e.g., [Explore view](https://docs.lumigo.io/docs/explore)) and alerting purposes.
+
+#### Creating Execution Tags
+
+In the Lumigo OpenTelemetry Distro for Java, execution tags are represented as [span attributes](https://opentelemetry.io/docs/reference/specification/common/#attribute) and, specifically, as span attributes with the `lumigo.execution_tags.` prefix.
+For example, you could add an execution tag as follows:
+
+```java
+import io.opentelemetry.api.trace.Span;
+
+Span.current().setAttribute("lumigo.execution_tags.foo","bar");
```

Comment:
What happens when you invoke Span.current() and there is currently no trace context?

Reply:
According to the code it's a no-op, but I will verify

Reply:
Verified that there are no NPEs when the distro is disabled